### PR TITLE
resultsdb plugin

### PR DESCRIPTION
**Pull request by apahim**
Propagate tests results to resultsdb.

Reference: https://trello.com/c/AcruJdgS/906-result-plugin-for-resultsdb
Signed-off-by: Amador Pahim <apahim@redhat.com>

### CODE REVIEW

**Comment by apahim** [comment]
[Result plugin for ResultsDB](https://trello.com/c/AcruJdgS/906-result-plugin-for-resultsdb)

File: optional_plugins/resultsdb/avocado_resultsdb/__init__.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            data = {'time_elapsed': "%.2f s" % test['time_elapsed'],
+                    'time_start': time.strftime("%Y-%m-%d %H:%M:%S",
+                                                local_time_start),
+                    'time_end': time.strftime("%Y-%m-%d %H:%M:%S",
+                                              local_time_end),
+                    'logdir': test['logdir'],
+                    'logfile': test['logfile'],
+                    'whiteboard': test['whiteboard']}
+
+            rdbapi.create_result(outcome,
```
Comment (clebergnu) [inline optional_plugins/resultsdb/avocado_resultsdb/__init__.py]: Super nice!

File: optional_plugins/resultsdb/avocado_resultsdb/__init__.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    description = "Resultsdb options for 'run' subcommand"
+
+    def configure(self, parser):
+        run_subcommand_parser = parser.subcommands.choices.get('run', None)
+        if run_subcommand_parser is None:
+            return
+
+        msg = 'resultsdb options'
+        parser = run_subcommand_parser.add_argument_group(msg)
+        parser.add_argument('--resultsdb-api',
```
Comment (clebergnu) [inline optional_plugins/resultsdb/avocado_resultsdb/__init__.py]: I'd also go with a configuration entry, it seems like the type of thing that you'd always point to the same URL.

Comment (apahim) [inline optional_plugins/resultsdb/avocado_resultsdb/__init__.py]: Sure, makes sense.

**Pull request closed without merging**